pkg/plugin/test-keeper/plugin/test_fixtures/github_calls/prs/without_tests/deletions_only_changes_in_tests.json

```diff
@@ -0,0 +1,37 @@
+    },
+    {
+      "sha": "fdeb66e6c0f3c09564dc2297966af3ae847b52be",
+      "filename": "src/test/java/io/openshift/booster/NewGreetingTest.java",
+      "status": "removed",
+      "additions": 0,
+      "deletions": 0,
+      "changes": 0,
+      "blob_url": "https://github.com/bartoszmajsak/wfswarm-booster-pipeline-test/blob/7a4d81ec7579fa508ef0abd97c5c5141b0fe5740/src/test/java/io/openshift/booster/NewGreetingTest.java",
+      "raw_url": "https://github.com/bartoszmajsak/wfswarm-booster-pipeline-test/raw/7a4d81ec7579fa508ef0abd97c5c5141b0fe5740/src/test/java/io/openshift/booster/NewGreetingTest.java",
+      "contents_url": "https://api.github.com/repos/bartoszmajsak/wfswarm-booster-pipeline-test/contents/src/test/java/io/openshift/booster/NewGreetingTest.java?ref=7a4d81ec7579fa508ef0abd97c5c5141b0fe5740"
+    }
+ ] 
```

Comment:
I would suggest making those files based on actual PRs with intended content. Here's a the real one:

```json
[
  {
    "sha": "eaed6f6c17eca636fd2ac190001495248f053f39",
    "filename": "src/test/java/io/openshift/booster/GreetingServiceTest.java",
    "status": "modified",
    "additions": 0,
    "deletions": 13,
    "changes": 13,
    "blob_url": "https://github.com/bartoszmajsak/wfswarm-booster-pipeline-test/blob/0beab1ffd081660b0892764d2cdfcdfb0e9ea8e3/src/test/java/io/openshift/booster/GreetingServiceTest.java",
    "raw_url": "https://github.com/bartoszmajsak/wfswarm-booster-pipeline-test/raw/0beab1ffd081660b0892764d2cdfcdfb0e9ea8e3/src/test/java/io/openshift/booster/GreetingServiceTest.java",
    "contents_url": "https://api.github.com/repos/bartoszmajsak/wfswarm-booster-pipeline-test/contents/src/test/java/io/openshift/booster/GreetingServiceTest.java?ref=0beab1ffd081660b0892764d2cdfcdfb0e9ea8e3",
    "patch": "@@ -50,17 +50,4 @@ public void test_service_invocation() {\n         Assert.assertTrue(response.readEntity(String.class).contains(\"Hello, World!\"));\n     }\n \n-    @Test\n-    @RunAsClient\n-    public void test_service_invocation_withParam() {\n-        Client client = ClientBuilder.newClient();\n-        WebTarget target = client.target(\"http://localhost:8080\")\n-                .path(\"api\").path(\"greeting\")\n-                .queryParam(\"name\", \"Peter\");\n-\n-        Response response = target.request(MediaType.APPLICATION_JSON).get();\n-        Assert.assertEquals(200, response.getStatus());\n-        Assert.assertTrue(response.readEntity(String.class).contains(\"Hello, Peter!\"));\n-    }\n-\n }"
  }
]
```

Reply:
Also, let's name this file `deletions_only_changes_in_tests.json`

Reply:
And then the test case should be titled:
```golang
It("should block newly created pull request when deletions in the tests are the only changes" ...
```